src/io.jl

```diff
@@ -11,7 +11,8 @@ This function is similar to `Base.unsafe_read` but is different in some points:
 - It returns the number of bytes written to `output`.
 - It does not block if there are buffered data in `input`.
 """
-function unsafe_read(input::IO, output::Ptr{UInt8}, nbytes::Int)::Int
+function unsafe_read(input::IO, output::Ptr{UInt8}, nbytes::Integer)::Int
```

Comment:
Another oddity is the type annotation `Integer)::Int`, since the return type is converted to `Int`, that seems redundant

Reply:
Done